### DISCUSSION TITLE
Option to hide front-matter description in pages?

### DISCUSSION THREAD

**Discussion by gmichaelides** [google/docsy#1706] [CSS and other site hacks]
Is there a preferred way to hide the front-matter description from the pages themselves?

For example, page https://www.docsy.dev/docs/adding-content/lookandfeel/ shows under the heading:

"Customize colors, fonts, code highlighting, and more for your site."

Which it derives from front-matter,

description: "Customize colors, fonts, code highlighting, and more for your site."

Adding the following in _variables_project.scss seems to work but wanted to check if there is an alternative.

/* Hide description in pages */
.td-content .lead {
    display: none;
}

**Comment by fekete-robert** [comment]
Hi, it seems to me that currently there is no other option to disable that (https://github.com/search?q=repo%3Agoogle%2Fdocsy%20class%3D%22lead%22&type=code).

**Reply by gmichaelides** [reply]
Thank you, this was helpful.

It seems that we could define a parameter and then an if statement by modifying something like list.html to display or not.

In this example, adding a parameter in frontmatter and {{ if .Params.show_lead }} may work, although a global parameter could be a better option.
...
show_lead: true
...

<div class="td-content">
	<h1>{{ .Title }}</h1>
	{{ if .Params.show_lead }}
		{{ with .Params.description }}<div class="lead">{{ . | markdownify }}</div>{{ end }}
	{{ end }}
	<header class="article-meta">

Thanks again.
